common/cairo.c

```diff
@@ -101,7 +92,9 @@ cairo_surface_t* gdk_cairo_image_surface_create_from_pixbuf(const GdkPixbuf *gdk
 		 * ------
 		 * tested as equal to lround(z/255.0) for uint z in [0..0xfe02]
 		 */
-#define PREMUL_ALPHA(x,a,b,z) G_STMT_START { z = a * b + 0x80; x = (z + (z >> 8)) >> 8; } G_STMT_END
+#define PREMUL_ALPHA(x,a,b,z) \
+		G_STMT_START { z = a * b + 0x80; x = (z + (z >> 8)) >> 8; } \
+		G_STMT_END
 		int i;
 		for (i = h; i; --i) {
 			const guint8 *gp = gdkpix;
```

Comment:
This file contains formatting fixes but also addresses a long-time TODO in Sway 0.x to bring the cairo helpers into libsway-common so we could use them in both clients and the compositor